app/javascript/src/components/Dashboard/Notes/Collection/Card/Footer.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+import { Clock } from "neetoicons";
+import { Avatar, Tag, Typography, Tooltip } from "neetoui";
+import { useTranslation } from "react-i18next";
+
+import { getDateWeekTime, getElapsedTime } from "utils/dateFormat";
+
+import { DEFAULT_USER_AVATAR } from "../../constants";
+
+const Footer = ({ createdAt }) => {
+  const { t } = useTranslation();
+
+  return (
+    <div className="flex flex-row justify-between">
+      <Tag label={t("common.gettingStarted")} style="secondary" type="solid" />
+      <div className="flex items-center gap-1">
+        <Clock size={12} />
+        <Tooltip content={getDateWeekTime(createdAt)} position="bottom-end">
```

Comment:
You are not `get`ting anything here. Rather you are formatting the already existing date. So name this something like `formatToDateWeekTime`